elasticsearch/autocomplete/src/test/java/nl/knaw/huc/resources/AutocompleteResourceTest.java

```diff
@@ -108,6 +108,26 @@ public void testFields_returns422UnprocessableEntity_whenPdf() throws IOExceptio
     assertThat(fields).contains("Unexpected mimetype: got [application/pdf] but should be one of [");
   }
 
+  @Test
+  public void testFields_returnsCompletionSuggesterInput_whenPageXml() throws IOException {
+    var fileContents = getResourceAsBytes("file.page.xml");
+    var response = postTestContents(fileContents, "application/vnd.prima.page+xml");
+    assertThat(response.getStatus()).isEqualTo(200);
+    var fields = response.readEntity(String.class);
+    var firstInput = JsonPath.parse(fields).read("$.suggest[0].input");
+    assertThat(firstInput).isEqualTo("evolutie");
+    var firstWeight = JsonPath.parse(fields).read("$.suggest[0].weight");
+    assertThat(firstWeight).isEqualTo(1);
+  }
+
+  @Test
+  public void testTypes_returnsMimetypesAndSubtypes() throws IOException {
+    var response = application.client().target(getTestUrl("/types")).request().get();
+    var types = response.readEntity(String.class);
+    assertThat(types).isEqualTo("{\"types\":[{\"mimetype\":\"application/xml\",\"subtypes\":[\"application/vnd.prima" +
```

Comment:
Ik zou de string split niet midden in een mimetype doen.
Liever het hele `"application/vnd.prima.page.xml"` in 1 String als het kan.

Reply:
19fd0c2